### ISSUE TITLE
There is nothing here.

### ISSUE THREAD

**Issue by hanss314** [carykh/EvolutionSimulator2D#1]
Where is Evolution Simulator?

**Comment by hanss314** [comment]
Hello? Cary?

**Comment by morgandonze** [comment]
I found it. Its [here](https://www.openprocessing.org/sketch/377698)

**Comment by hanss314** [comment]
the repo is still empty

**Comment by morgandonze** [comment]
Yes, but you can get the code at the link (click the `</>` icon)

**Comment by hanss314** [comment]
then what's the point of this repo?

**Comment by morgandonze** [comment]
I dunno :rofl:

**Comment by madrang** [comment]
Got the source from here https://drive.google.com/drive/folders/0B98DkR2AsGZaNzlGSFlXaHF2NG8
Fixed it to work with processing 3.*

I also properly implemented multi threading and working on fixing a few things with the colliders, but it's working and faster now.

But i can't do a pull request on an empty git repository....

**Comment by madrang** [comment]
![selection_517](https://user-images.githubusercontent.com/747243/36246177-84eec18c-11fc-11e8-80a2-58ebb636adee.jpg)

![selection_518](https://user-images.githubusercontent.com/747243/36246215-b14bc2b6-11fc-11e8-941d-bd6cca564e82.jpg)

Edit: I think they did something weird by year 563.....
https://imgur.com/a/JYM4o

**Comment by madrang** [comment]
Now using a hashgrid for collision detection and an improved brain analyser.
Also a few other changes here and there. Would love to do a git clone before i submit my version to github as it would show that i started with code from here.

![selection_019](https://user-images.githubusercontent.com/747243/36406375-4378341a-15c4-11e8-93a0-600fc5858819.png)
![selection_020](https://user-images.githubusercontent.com/747243/36406387-4dcbc7ec-15c4-11e8-9c1c-24d46b5dafa5.png)
![evolviocrunch_018](https://user-images.githubusercontent.com/747243/36406389-568e7154-15c4-11e8-90ce-268198ae079b.png)
In it shows constants in red, active in light blue and positive as green and negative as purple.

**Comment by vgzhn** [comment]
@madrang are you able to share your fixed version?

**Comment by madrang** [comment]
@vgzhn It's up here https://github.com/madrang/EvolvioCrunch
